repo/packages/P/portworx-prometheus/0/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "packagingVersion": "4.0", 
+    "upgradesFrom": [
+        "*"
+    ], 
+    "downgradesTo": [
+        "*"
+    ], 
+    "minDcosReleaseVersion": "1.10", 
+    "name": "portworx-prometheus", 
+    "version": "0.1.1-2.3.2", 
```

Comment:
Copying @disrani-px's comment - 
Needs to include our sdk version similar to other services